tensorizer/serialization.py

```diff
@@ -3984,11 +3999,12 @@ def write_state_dict(self, state_dict: Dict):
 
 
 def _get_perf_stats():
-    return dict(
-        cuda_to_device_secs=float(_perf_stats.cuda_to_device_millisecs.load())
-        / 1000.0,
-        cuda_bytes=_perf_stats.cuda_bytes.load(),
-        file_readinto_secs=float(_perf_stats.file_readinto_millisecs.load())
-        / 1000.0,
-        file_readinto_bytes=_perf_stats.file_readinto_bytes.load(),
-    )
+    if _perf_stats is None:
+        raise RuntimeError("Performance stats are not enabled")
```

Comment:
Maybe mention how this can be enabled by setting `TENSORIZER_ENABLE_PERF_STATS"` in the `RuntimeError`.

Reply:
Since it is meant to be internal-only, I'm not sure we want to guide anyone on how to enable it unless they already know what they are doing with it (though on that same note, it should probably not be accessed in the `deserialize.py` example meant for public use, but rather something in the benchmarks folder). A source code comment next to the exception could be a compromise, though. What do you think?

Reply:
I'm fine moving the current `deserialize.py` to `benchmarks/`, just as long as it stays intact :)